src/content/contributions/component/components.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Components
+---
+
+### Component contributions can take several forms. Most are either component enhancements or brand new assets.
+
+## First steps to contributing
+
+To contribute a component to Carbon, start by [opening a Github issue](https://github.com/carbon-design-system/carbon/issues/new?assignees=&labels=type%3A+enhancement+%F0%9F%92%A1&template=feature-request-or-enhancement.md&title=). Include a detailed description in which you:
+* Explain the rationale
+* Detail the intended behavior
+* Clarify whether it's a variation of an existing component, or a new asset
+* Include mockups of any fidelity (optional)
+* Include any inspirations from other products (optional)
+
+This issue will be the staging ground for the contribution and an opportunity for the community to weigh in with any suggestions. We'll also be able to tell you up-front whether the contribution is likely to become part of Carbon, and potentially save you time. 
+
+We encourage you to surface works-in-progress. If you're not able to complete all of the parts yourself, someone in the community may be able to pick up where you leave off.
+
+## Parts of a component contribution
+
+Here are all of the steps for a complete contribution. 
+
+#### 1. Provide a rationale
+
+Explain how your component will add value to the system. Carbon serves the widest possible range of products, and contributions that increase the scope of the system are more likely to be accepted. Be sure to include any user experience and interaction descriptions.
+
+#### 2. Create a design spec
+
+Create sizing and styling annotations for all aspects of the component. This spec should provide a developer with everything they need to create the design in code. Check out our [production guidelines](https://github.com/carbon-design-system/carbon-website/wiki/Production-guidelines#spec-guidelines) to get started.
+
+You should include color tokens and type tokens used.
+
+<ImageComponent caption="Example of a spec" cols="8">
+
+![Example of a spec](images/spec.png)
```

Comment:
This example is a little off, you wouldn't need that 2px spec which is measuring the spec not the component. Not a blocker to the PR but we could eventually drop a better example in here. Actually you could probably just pull an image from one of the component/style tabs as a good example.

Reply:
Will do. I just pulled this one from the production guidelines.